app/src/main/java/edu/uc/zhukv/droneradarmap/ui/home/HomeFragment.kt

```diff
@@ -1,4 +1,5 @@
 package edu.uc.zhukv.droneradarmap.ui.home
 
```

Comment:
Good rule of thumb is to put a TODO in the file so that you don't forget to implement a file. 

Rationale: Seemed like a future implementation so I created a TODO to prevent the creation of redundant files which has been evident throughout this project.

Reply:
Better rule of thumb is just to finish it.  TODOs are clutter.

1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.